tests/bsim/bluetooth/audio/src/csip_set_member_test.c

```diff
@@ -76,6 +76,35 @@ static void bt_ready(int err)
 	}
 }
 
+static void test_set_sirk(void)
+{
+	const uint8_t new_set_sirk[] = {0xff, 0xcc, 0x72, 0xdd, 0x86, 0x8c, 0xcd, 0xce,
+					0x22, 0xfd, 0xa1, 0x21, 0x09, 0x7d, 0x7d, 0x45};
+	uint8_t tmp_sirk[BT_CSIP_SET_SIRK_SIZE];
+	int err;
+
+	printk("Setting new SIRK\n");
+	err = bt_csip_set_member_set_sirk(svc_inst, new_set_sirk);
+	if (err != 0) {
+		FAIL("Failed to set SIRK: %d\n", err);
+		return;
+	}
+
+	printk("Getting new SIRK\n");
+	err = bt_csip_set_member_get_sirk(svc_inst, tmp_sirk);
+	if (err != 0) {
+		FAIL("Failed to set SIRK: %d\n", err);
```

Comment:
Not going to stress on typos in test code... (not reviewing further...)

```suggestion
		FAIL("Failed to get SIRK: %d\n", err);
```